sources/install/package_base.sh

```diff
@@ -396,6 +396,16 @@ function post_install() {
     if [ -d "/root/Desktop" ]; then rm -r /root/Desktop; fi
 }
 
+function uploader() {
+    colorecho "Installing Uploader"
+    git clone https://github.com/Frozenka/uploader.git /opt/tools/uploader
```

Comment:
Can you add `-C /opt/tools` and `--depth 1` ? (`git -C /opt/tools/ clone --depth 1 https://.....`)